src/snowflake/snowpark/session.py

```diff
@@ -2749,4 +2749,32 @@ def _explain_query(self, query: str) -> Optional[str]:
             _logger.warning("query `%s` cannot be explained", query)
             return None
 
+    @staticmethod
+    def create() -> "Session":
```

Comment:
Is `create` the right action name for "gets existing session if one exists", or should this be `fetch` instead?

Reply:
I think there is already a method called `getOrCreate`

Reply:
So basically this comes from a request from field/PM: https://docs.google.com/document/d/12FGgc9NeH-xa31FMZa_lO2N5Xq71Cm0aquTLI2-G0IE/edit#heading=h.hvtde6rw3c3p

They want a unified `connect` method in session (which I mistakenly named the method `created`) which works across different use-cases (i.e., when Snowpark is used as is or in SiS, SP, SPCS)

Reply:
I see. @sfc-gh-mkeller @sfc-gh-yixie : Do you have concerns for introducing a `connect` method in session?